.github/CODEOWNERS

```diff
@@ -28,14 +31,16 @@
 /corehq/apps/commtrack/ @esoergel
 /corehq/apps/commtrack/static/ @esoergel @orangejenny
 /corehq/apps/commtrack/templates/ @esoergel @orangejenny
-/corehq/apps/couch_sql_migration/ @snopoke @emord
+/corehq/apps/couch_sql_migration/ @snopoke @emord @millerdev
 /corehq/apps/es/ @esoergel
 /corehq/apps/hqwebapp/ @orangejenny
 /corehq/apps/locations/ @esoergel
+/corehq/apps/locations/adjacencylist.py @esoergel @millerdev
 /corehq/apps/locations/static/ @esoergel @orangejenny
 /corehq/apps/locations/templates/ @esoergel @orangejenny
 /corehq/apps/ota/ @snopoke
 /corehq/apps/receiverwrapper/ @snopoke
+/corehq/apps/sms*/ @millerdev
```

Comment:
If this rule is saying you want to get notifications on sms, smsforms, and smsbillables, I think you need to add yourself to the line below to get smsbillables.

Reply:
No, I want ownership of anything `.../sms*` that is not claimed by someone else